docs/admin/systemd.md

```diff
@@ -33,15 +33,20 @@ If you want Docker to start at boot, you should also:
 There are a number of ways to configure the daemon flags and environment variables
 for your Docker daemon.
 
-The recommended way is to use a systemd drop-in file. These are local files in
-the `/etc/systemd/system/docker.service.d` directory. This could also be
-`/etc/systemd/system/docker.service`, which also works for overriding the
-defaults from `/lib/systemd/system/docker.service`.
-
-However, if you had previously used a package which had an `EnvironmentFile`
-(often pointing to `/etc/sysconfig/docker`) then for backwards compatibility,
-you drop a file in the `/etc/systemd/system/docker.service.d`
-directory including the following:
+The recommended way is to use a systemd drop-in file (as described in
+the [systemd.unit][] documentation). These are local files named
```

Comment:
Oh! You're right;  actually, we don't use referenced URL's like that in our docs, so I'd prefer them inline to be consistent.

(I agree that the plain HTML links are a bit quirky, we're looking into alternative options for the tooling we use, but for now they do the job)